torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1223,6 +1223,83 @@ that adds a prefix to each key inserted to the store.
 
 #undef PROCESS_GROUP_DEPRECATION_WARNING
 
+// NOTE: Below are TorchBind bindings for c10d, these bindings will
+// live together with those pybind11 bindings above until we resolve
+// all the TorchBind issues and merge these two together. we shouldn't
+// document this until we finish the migration.
+
+static const auto StoreTorchBind =
+    torch::class_<::c10d::Store>("dist_c10d", "Store");
```

Comment:
(not requesting for change)

If `dist_c10d` is what users will see when this is done, let's try to come up with a better name for it. Maybe, sth like `torch.classes.distributed` or `torch.classes.distributed.comm`, etc.

Reply:
yes I agree with that, the naming is bad. That's something torchbind didn't support yet, we will be adding that support soon and I will switch the names over in follow up PRs.